aayushi/src/pages/ForgotPassword.jsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+import { z } from "zod";
+import { Button } from "../components/Button";
+import { Input } from "../components/Input";
+import { Label } from "../components/Label";
+
+const forgotPasswordSchema = z.object({
+  email: z.string().email("Must be a valid email address"),
+});
+
+export const ForgotPassword = ({ setPassword }) => {
+  const [data, setData] = useState({
+    email: "",
+  });
+  const [error, setError] = useState({});
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const result = forgotPasswordSchema.safeParse(data);
+
+    if (result.success) {
+      setLoading(true);
+      console.log("Email submitted successfully:", data.email);
+      setError({});
+
+      setTimeout(() => {
+        setLoading(false);
+      }, 2000);
+    } else {
+      const errorMessages = {};
+      result.error.errors.forEach((err) => {
+        errorMessages[err.path[0]] = err.message;
+      });
+      setError(errorMessages);
+    }
+  };
+
+  return (
+    <div className="font-poppins">
+      <div className="p-8 ">
+        <p className="text-xl font-semibold text-center">
+          Forgot Your Password?
+        </p>
+      </div>
+      <div className="px-6 mt-4 mb-36">
+        <p className="text-sm text-center">
```

Comment:
Here both the child elements of this div need the property of text centre so mention it directly in the parent div.